tests/command_test.c

```diff
@@ -145,13 +145,14 @@ main (int argc, char *argv [])
 		} ;
 
 	if (do_all || strcmp (argv [1], "cue") == 0)
-	{	int i, cuecounts[] = {-1, 0, 1, 100, 101, 255, 256, 1000};
+	{	int cuecounts[] = {-1, 0, 1, 100, 101, 1000}; /* 1000 is the largest number of cues possible */
+		int i, n = ARRAY_LEN(cuecounts);
 	    
-		cue_test ("cue.wav", SF_FORMAT_WAV | SF_FORMAT_PCM_16) ;
+		cue_test ("cue.wav",   SF_FORMAT_WAV  | SF_FORMAT_PCM_16) ;
 		cue_test ("cue.aiff" , SF_FORMAT_AIFF | SF_FORMAT_PCM_24) ;
 
-		for (i = 0; i < (int) sizeof(cuecounts) / sizeof(int); i++)
-		{	cue_test_var ("cue.wav",  SF_FORMAT_WAV | SF_FORMAT_PCM_16,  cuecounts[i]) ;
+		for (i = 0; i < n; i++)
```

Comment:
I'd rather see `ARRAY_LEN(cuecounts)` there (it serves as documentaion) than `'n` so I don't have to search the code for the definition of `n`.